### PR TITLE
Switch rules instead of adding a second prt node

### PR DESCRIPTION
**Pull request by chr11115**
- Check for an existing serlio node and switch the rule package instead of using a complex building as the new initial shape for another serlio node. (only works with active construction history)
- Now behaves like switching an rpk on the serlionode itself

### CODE REVIEW

File: src/serlio/scripts/serlioCreateUI.mel [hunk 1]
```diff
@@ -51,10 +51,27 @@ global proc createPrtNode() {
 	string $rulePackage[] = `fileDialog2 -fm 1 -cap "Select Rule Package" -ff $filters`;
 	
 	
+	int $alreadyHasExistingPrtNode = false;
+
 	if(size($rulePackage)) {
```
Comment (mistafunk) [inline src/serlio/scripts/serlioCreateUI.mel:56]: Similar as in #89, consider writing the explicit `>0` check here as well

File: src/serlio/scripts/serlioCreateUI.mel [hunk 2]
```diff
@@ -51,10 +51,27 @@ global proc createPrtNode() {
...
+			for($dpNode in $nodeHistory){
+				$dpNodeType = `nodeType $dpNode`;
+				if($dpNodeType == "serlio"){
+					setAttr -type "string" ($dpNode + ".Rule_Package") $rulePackage[0];  
+					evalDeferred "refreshEditorTemplates()";
+					$alreadyHasExistingPrtNode = true;
+				}
+			}
+
+			if (!alreadyHasExistingPrtNode){
```
Comment (mistafunk) [inline src/serlio/scripts/serlioCreateUI.mel:70]: Missing `$` character, I think.

// Error: "F:/dev/git/serlio/install/scripts/serlioCreateUI.mel" line 70.34: Invalid use of Maya object "alreadyHasExistingPrtNode". //

**Review by mistafunk — CHANGES_REQUESTED** [review]
Small syntax error in the script.

**Comment by chr11115** [comment]
added type definition for string

**Review by mistafunk — APPROVED** [review]
LGTM, verified functionality with Maya 2020 on LInux. Feel free to merge.

**Pull request merged**